test/zdtm/static/epoll01.c

```diff
@@ -0,0 +1,119 @@
+#include <unistd.h>
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <sys/eventfd.h>
+#include <sys/ioctl.h>
+#include <sys/epoll.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <inttypes.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc	= "Check for epoll";
```

Comment:
Could you add a comment that explains what this test checks? It can be the description from the commit message.

Reply:
I've already put to commit message:
```
This adds three epoll targets on tfd 702 and then adds two epoll targets
on tfd 701. This test is for off calculation in dump_one_eventpoll, the
reverse order makes qsort to actually work.
```
Isn't this enough?

Reply:
After rereading your message once again, look like you just ask to add description as a comment in code. Sure - no problem. Done.